### ISSUE TITLE
Problems with X3D training results

### ISSUE THREAD

**Issue by Hao-xin-Lu** [facebookresearch/SlowFast#686]
I used x3d pre-trained kinematics 400 and trained 8epochs, in the result, top-1 accuracy:99.8,top-5 accuracy:0.00.
Does anyone know why?

TRAIN:
  ENABLE: True
  DATASET: ava
  BATCH_SIZE: 8
  EVAL_PERIOD: 10
  CHECKPOINT_PERIOD: 10
  AUTO_RESUME: True
  CHECKPOINT_FILE_PATH: 'C:/Users/name/slowfast/x3d/test/x3d_l.pyth'
  CHECKPOINT_TYPE: pytorch
DATA:
  NUM_FRAMES: 16
  SAMPLING_RATE: 5
  TRAIN_JITTER_SCALES: [256, 320]
  TRAIN_CROP_SIZE: 224
  TEST_CROP_SIZE: 224
  INPUT_CHANNEL_NUM: [3]
  DECODING_BACKEND: torchvision
  PATH_TO_DATA_DIR: 'D:/Download/AVA'
DETECTION:
  ENABLE: True
  ALIGNED: True
  SPATIAL_SCALE_FACTOR: 32
AVA:
  DETECTION_SCORE_THRESH: 0.8
  TRAIN_PREDICT_BOX_LISTS: [
  "ava_train_v2.2.csv",
  "person_box_67091280_iou90/ava_detection_train_boxes_and_labels_include_negative_v2.2.csv",
  ]
  TEST_PREDICT_BOX_LISTS: ["person_box_67091280_iou90/ava_detection_val_boxes_and_labels.csv"]
  FRAME_DIR: 'D:/Download/AVA/frames'
  FRAME_LIST_DIR: 'D:/Download/AVA/frame_lists'
  ANNOTATION_DIR: 'D:/Download/AVA/annotations'
X3D:
  WIDTH_FACTOR: 2.0
  DEPTH_FACTOR: 5.0
  BOTTLENECK_FACTOR: 2.25
  DIM_C5: 2048
  DIM_C1: 12
RESNET:
  ZERO_INIT_FINAL_BN: True
  TRANS_FUNC: x3d_transform
  STRIDE_1X1: False
BN:
  USE_PRECISE_STATS: False
  NUM_BATCHES_PRECISE: 200
  WEIGHT_DECAY: 0.0 
SOLVER:
  BASE_LR: 0.1
  BASE_LR_SCALE_NUM_SHARDS: True
  LR_POLICY: steps_with_relative_lrs
  STEPS: [0, 10, 15, 20]
  LRS: [1, 0.1, 0.01, 0.001]
  MAX_EPOCH: 20
  WEIGHT_DECAY: 1e-7
  WARMUP_EPOCHS: 5.0
  WARMUP_START_LR: 0.000125
  OPTIMIZING_METHOD: sgd
MODEL:
  NUM_CLASSES: 80
  ARCH: x3d
  MODEL_NAME: X3D
  LOSS_FUNC: bce
  DROPOUT_RATE: 0.5
  HEAD_ACT: sigmoid
TEST:
  ENABLE: False
  DATASET: ava
  BATCH_SIZE: 1
DATA_LOADER:
  NUM_WORKERS: 0
  PIN_MEMORY: True
NUM_GPUS: 1
NUM_SHARDS: 1
RNG_SEED: 0
OUTPUT_DIR: ./x3d
